curriculum/challenges/english/25-front-end-development/review-javascript-maps-and-sets/6723d027b02e4cc6ee5944da.md

```diff
@@ -26,6 +26,8 @@ console.log(set); // Set { 1, 2, 3, 4, 5 }
   - `delete()`: Removes an element from the `Set`.
   - `has()`: Checks if an element exists in the `Set`.
   - `clear()`: Removes all elements from the `Set`.
+  - `keys()` and `values()`: Both returens a `SetIterator` that contains the values of the `Set`. They are the same because `keys()` is an alias for `values()`.
```

Comment:
```suggestion
  - `keys()` and `values()`: Both returns a `SetIterator` that contains the values of the `Set`. They are the same because `keys()` is an alias for `values()`.
```